.github/CODEOWNERS

```diff
@@ -137,7 +139,11 @@ CHANGELOG*
 /x-pack/metricbeat/ @elastic/elastic-agent-data-plane
 /x-pack/metricbeat/docs/ # Listed without an owner to avoid maintaining doc ownership for each input and module.
 /x-pack/metricbeat/module/ @elastic/integrations
+/x-pack/metricbeat/module/containerd/ @elastic/obs-cloudnative-monitoring
 /x-pack/metricbeat/module/enterprisesearch @elastic/ent-search-application-backend
+/x-pack/metricbeat/module/istio/ @elastic/obs-cloudnative-monitoring
+/x-pack/metricbeat/module/prometheus/ @elastic/obs-cloudnative-monitoring
+/x-pack/metricbeat/module/stan/ @elastic/obs-cloudnative-monitoring
```

Comment:
Also the `nats` module should be added?

Reply:
@ChrsMark added https://github.com/elastic/beats/pull/34128/commits/fcc7c39ebfc510487081b43369945fc5473d875a